client/src/pages/Landing/Landing.js

```diff
@@ -53,20 +76,45 @@ const Landing = () => {
 
   return (
     <View style={styles.root}>
-      <StatusBar barStyle="light-content" />
-      <Text
-        fontWeight="regular"
-        color="blue.dark"
-        fontStyle="italic"
-        fontSize="6xl"
-      >
-        Landing
-      </Text>
-      <StyledButton
-        title="Login to app"
-        variant="primary"
-        onPress={loginUser}
-      />
+      <Image source={images.background_landing}></Image>
```

Comment:
I think this could use an alt attribute.

So `<Image ... alt = "some text describing the image" ...>`